lib/test.js

```diff
@@ -4,14 +4,19 @@ var setImmediate = require('set-immediate-shim');
 var fnName = require('fn-name');
 var assert = require('./assert');
 
-function Test(title, fn) {
+function Test(title, assertModule, fn) {
```

Comment:
`title` and `fn` are more common properties, while `assertModule` is absolutely optional. I think it needs to be the last one in the argument list.